pkg/algorithm/cstorpoolselect/v1alpha1/select.go

```diff
@@ -527,10 +703,11 @@ func FilterPoolIDs(entries *csp.CSPList, opts []buildOption) ([]string, error) {
 // go template functions
 func TemplateFunctions() template.FuncMap {
 	return template.FuncMap{
-		"cspGetPolicies":        GetPolicies,
-		"cspFilterPoolIDs":      FilterPoolIDs,
-		"cspAntiAffinity":       AntiAffinityLabel,
-		"cspPreferAntiAffinity": PreferAntiAffinityLabel,
-		"preferScheduleOnHost":  PreferScheduleOnHostAnnotation,
+		"cspGetPolicies":          GetPolicies,
+		"cspFilterPoolIDs":        FilterPoolIDs,
+		"cspAntiAffinity":         AntiAffinityLabel,
+		"cspPreferAntiAffinity":   PreferAntiAffinityLabel,
+		"preferScheduleOnHost":    PreferScheduleOnHostAnnotation,
+		"capacityAwareScheduling": CapacityAwareScheduling,
```

Comment:
it doesn't seem to really doing any scheduling (of replicas), but, provisioning PVs.. better to change the name to 'honorOverProvisioningFlag' or 'capacityAwareProvisioning'

Reply:
sure.